internal/command/limits.go

```diff
@@ -27,32 +29,83 @@ func (c *Commands) SetLimits(
 	if err != nil {
 		return nil, err
 	}
-	aggregateId := wm.AggregateID
-	if aggregateId == "" {
-		aggregateId, err = c.idGenerator.Next()
+	cmds, err := c.setLimitsCommands(ctx, wm, setLimits)
```

Comment:
as limits are now instance limits only, we should change the setLimits as well

the `resourceOwner` passed here is the instanceID of the (system api) request, so i guess we should use that and ignore the authz.GetInstance, right?

Reply:
done